pdfjs.config

```diff
@@ -1,6 +1,6 @@
 {
   "betaVersion": "",
```

Comment:
Just asking, but shouldn't there have been a beta version?

Also, what's up with the version number difference at https://github.com/mozilla/pdf.js/releases (i.e. `v2.1.245`) vs https://github.com/mozilla/pdfjs-dist/releases (i.e. `v2.2.2`).

Reply:
I commented about this on #10416. Basically, the instructions page wasn't really clear and I missed a step. I updated the page so that the next release will happen correctly again; sorry for the confusion!